src/deluge/model/clip/instrument_clip.cpp

```diff
@@ -94,6 +97,16 @@ InstrumentClip::InstrumentClip(Song* song) : Clip(CLIP_TYPE_INSTRUMENT) {
 	inScaleMode = (FlashStorage::defaultScale != PRESET_SCALE_NONE);
 	onKeyboardScreen = false;
 
+	//initialize automation instrument clip view variables
+	onAutomationInstrumentClipView = false;
+	lastSelectedParamID = 255;
+	lastSelectedParamType = 255;
+	lastSelectedParamShortcutX = 255;
+	lastSelectedParamShortcutY = 255;
```

Comment:
I think I mentioned it above but all of these 255s should probably have names (`kNoLastXXX` probably, to suggest a color for the bike shed)